typings/index.d.ts

```diff
@@ -1,3 +1,20 @@
+declare enum ApplicationCommandOptionTypes {
```

Comment:
Why don't we import the enums from discord-api-types?

Reply:
Probably because of consistency (or because it hasnt been a dependency yet when this was added), I'll change that